src/lib.rs

```diff
@@ -964,6 +965,14 @@ impl<F: Read + Write + Seek> CompoundFile<F> {
     }
 }
 
+impl<F: fmt::Debug> fmt::Debug for CompoundFile<F> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_tuple("CompoundFile")
+            .field(self.minialloc().directory().allocator().sectors().inner())
```

Comment:
Rather than have all these accessors (and have to call down the chain here, which also means having to update this one random line if the tower of abstractions changes in the future), I'm more inclined to give each struct in the chain an `inner(&self) -> &F` method that defers to the next `inner()` in the chain, the way we already do with `into_inner()`.

Reply:
Should be all set!